### ISSUE TITLE
BatchNorm3d instead of BatchNorm2d?

### ISSUE THREAD

**Issue by ranabanik** [MASILab/SLANTbrainSeg#13]
The model encoder has batch normalization enabled in 2D(`nn.BatchNorm2d`) instead it should be 3D?

**Comment by BennettLandman** [comment]
I am closing the older bug reports as these were missed. We are now better tracking reports across the organization. Please re-open if this continues to be a blocker.
